charts/generic/experiments.yaml

```diff
@@ -1223,6 +1222,127 @@ spec:
       app.kubernetes.io/runtime-api-usage: "true" 
       app.kubernetes.io/version: latest
 
+---
+
+apiVersion: litmuschaos.io/v1alpha1
+description:
+  message: |
+    it injects http status code chaos inside the pod which modifies the status code of the response from the provided application server to desired status code provided by user and reverts after specified duration
+kind: ChaosExperiment
+metadata:
+  name: pod-http-status-code
+  labels:
+    name: pod-http-status-code
+    app.kubernetes.io/part-of: litmus
+    app.kubernetes.io/component: chaosexperiment
+    app.kubernetes.io/version: latest
+spec:
+  definition:
+    scope: Namespaced
+    permissions:
+      - apiGroups: 
+          - "" 
+          - "batch" 
+          - "apps" 
+          - "litmuschaos.io"
+        resources: 
+          - "jobs" 
+          - "pods" 
+          - "pods/log" 
+          - "events" 
+          - "deployments" 
+          - "replicasets" 
+          - "pods/exec" 
+          - "chaosengines" 
+          - "chaosexperiments" 
+          - "chaosresults"
+        verbs: 
+          - "create" 
+          - "list" 
+          - "get" 
+          - "patch" 
+          - "update" 
+          - "delete" 
+          - "deletecollection"
+    image: "litmuschaos/go-runner:latest"
+    imagePullPolicy: Always
+    args:
+    - -c
+    - ./experiments -name pod-http-status-code
+    command:
+    - /bin/bash
+    env:
+
```

Comment:
Can we add envs for AppNS, AppLabel, etc?

Reply:
The appns etc are provided in the chaosengine (not in chaosexperiment).

Reply:
The `AppNs`, `AppLabel` and `AppKind` are provided under the `appInfo` and not in `components/env` as we provide other env variables. This is consistent with all other charts. If there is a benefit of adding it in experiments.yaml as well, I think we will need to discuss with maintainers about such change.